blog/_posts/2016-07-15-sip-summary-minutes.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: blog
+post-type: blog
+by: Jorge Vicente Cantero
+title: A Reformed Scala Improvement Process
```

Comment:
Take either this one ^ or the one I suggested earlier. Both are fine. Could you update the commit with the new title?